src/app/editor.js

```diff
@@ -23,7 +23,9 @@ var css = csjs`
     width             : 100%;
   }
 `
-document.querySelector('#editor-container').className = css['editor-container']
+setTimeout(_ => {
```

Comment:
i see. that makes sense.
I will update the pull request today, with the refactoring work in it.
It doesn't need the `.init` method anymore, but i understand how this would solve the issue.